pkg/ocm/output/nodepools_test.go

```diff
@@ -46,20 +44,3 @@ var _ = Describe("PrintNodePoolReplicasInline", func() {
 	})
 
 })
-
-var _ = Describe("Validate management upgrade print output", func() {
```

Comment:
@hunterkepley I don't see this test being replicated. It looks like it's just being deleted?